contracts/manager/DelegatedManager.sol

```diff
@@ -0,0 +1,468 @@
+
+            operators.removeStorage(operator);
+
+            operatorAllowlist[operator] = false;
+
+            emit OperatorRemoved(operator);
+        }
+    }
+
+    /**
+     * ONLY OWNER: Add new asset(s) that can be traded to, wrapped to, or claimed
+     *
+     * @param _assets           New asset to add
+     */
+    function addAllowedAssets(address[] memory _assets) external onlyOwner {
+        _addAllowedAssets(_assets);
+    }
+
+    /**
+     * ONLY OWNER: Remove asset(s) so that it/they can't be traded to, wrapped to, or claimed
+     *
+     * @param _assets           Asset to remove
+     */
+    function removeAllowedAssets(address[] memory _assets) external onlyOwner {
+        for (uint256 i = 0; i < _assets.length; i++) {
+            address asset = _assets[i];
+
+            require(assetAllowlist[asset], "Asset not already added");
+
+            allowedAssets.removeStorage(asset);
+
+            assetAllowlist[asset] = false;
+
+            emit AllowedAssetRemoved(asset);
+        }
+    }
+
+    /**
+     * ONLY OWNER: Toggle useAssetAllowlist on and off. When false asset allowlist is ignored
+     * when true it is enforced.
+     *
+     * @param _useAssetAllowlist           Bool indicating whether to use asset allow list
+     */
+    function updateUseAssetAllowlist(bool _useAssetAllowlist) external onlyOwner {
+        useAssetAllowlist = _useAssetAllowlist;
+
+        emit UseAssetAllowlistUpdated(_useAssetAllowlist);
+    }
+
+    /**
+     * ONLY OWNER: Update percent of fees that are sent to owner
+     *
+     * @param _newFeeSplit           Percent in precise units (100% = 10**18) of fees that accrue to owner
+     */
+    function updateOwnerFeeSplit(uint256 _newFeeSplit) external onlyOwner {
+        require(_newFeeSplit <= PreciseUnitMath.preciseUnit(), "Invalid fee split");
+
+        ownerFeeSplit = _newFeeSplit;
+
+        emit OwnerFeeSplitUpdated(_newFeeSplit);
+    }
+
+    /**
+     * ONLY OWNER: Update address owner receives fees at
+     *
+     * @param _newFeeRecipient           Address to send owner fees to
+     */
+    function updateOwnerFeeRecipient(address _newFeeRecipient) external onlyOwner {
+        require(_newFeeRecipient != address(0), "Null address passed");
+
+        ownerFeeRecipient = _newFeeRecipient;
+
+        emit OwnerFeeRecipientUpdated(_newFeeRecipient);
+    }
+
+    /**
+     * ONLY METHODOLOGIST: Update the methodologist address
+     *
+     * @param _newMethodologist           New methodologist address
+     */
+    function setMethodologist(address _newMethodologist) external onlyMethodologist {
+        emit MethodologistChanged(methodologist, _newMethodologist);
+        methodologist = _newMethodologist;
+    }
+
+    /**
+     * ONLY OWNER: Update the SetToken manager address.
+     *
+     * @param _newManager           New manager address
+     */
+    function setManager(address _newManager) external onlyOwner {
```

Comment:
Since we use the SetToken address to track data on the global extensions we need to make sure that all extensions have been cleared here in order to allow a new manager to be initialized.

Reply:
Either need to add a require or cycle through and remove the extensions in this function